### DISCUSSION TITLE
Can you use MFA and or SSO with the wallet connect web app?

### DISCUSSION THREAD

**Discussion by justinengland** [WalletConnect/walletconnect-monorepo#1705] [Developer Support]
Hello! Recently signed up for a corporate account and I am curious what patterns are provided to protect these accounts outside of normal username and password login?

**Comment by chadyj** [comment]
Hey @justinengland. You can sign in with Ethereum to cloud.walletconnect.com as an alternative to email and password.

See this blog post for more info.

https://medium.com/walletconnect/walletconnect-cloud-you-can-now-sign-in-with-ethereum-7563578686c2
